net-mail/cmd5checkpw/cmd5checkpw-0.30-r2.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils fixheadtails
+
+MY_VER=$(ver_rs 1- "")
+
+DESCRIPTION="A checkpassword compatible authentication program that used CRAM-MD5 authentication mode"
+SRC_URI="https://www.fehcom.de/qmail/auth/${PN}-${MY_VER}_tgz.bin -> ${P}.tar.gz"
+HOMEPAGE="https://www.fehcom.de/qmail/smtpauth.html"
+
+LICENSE="public-domain RSA"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE=""
+
+DEPEND="acct-user/cmd5checkpw"
+RDEPEND="${DEPEND}"
+
+pkg_setup() {
+	ewarn
+	ewarn "this version is in NO WAY COMPATIBLE with cmd5checkpw-0.2x"
```

Comment:
Whats this message trying to convey here? Can this be automated in some ebuild, by using `has_version` for example?

Reply:
done.